### DISCUSSION TITLE
How to embedded youtube video?

### DISCUSSION THREAD

**Discussion by CAnorbo** [vuepress/core#1182] [Q&A] [answered]
Hello,

I would like to add a lot of embedded videos from youtube to my Vuepress 2.x site.
How should I do it?

**Comment by azat-io** [comment]
You can use html in your markdown files.

Just put something like this to you `.md` file:

```
<iframe width="560" height="315" src="https://www.youtube.com/embed/2bba5ldIgbw" title="YouTube video player" frameborder="0" allow="accelerometer; autoplay; clipboard-write; encrypted-media; gyroscope; picture-in-picture" allowfullscreen></iframe>
```

**Comment by Mister-Hope** [comment, marked as answer]
https://vuepress-theme-hope.github.io/v2/components/guide/youtube.html

here is a plugin which contains component you need

**Reply by CAnorbo** [reply]
Thank you!
